contextConsumption/query_entities_test.js

```diff
@@ -1,136 +1,149 @@
+const testedResource = require("../common.js").testedResource;
+const http = require("../http.js");
 
+const entitiesResource = testedResource + "/entities/";
+const assertRetrievedQuery = require("../common.js").assertRetrievedQuery;
+const serializeParams = require("../common.js").serializeParams;
 
-const testedResource = require('../common.js').testedResource;
-const http = require('../http.js');
-
-const entitiesResource = testedResource + '/entities/';
-const assertRetrievedQuery = require('../common.js').assertRetrievedQuery;
-const serializeParams = require('../common.js').serializeParams;
-
-describe('Query Entity. JSON. Default @context', () => {
+describe("Query Entity. JSON. Default @context", () => {
   const entity = {
-    'id': 'urn:ngsi-ld:T_Query:EntityForQuery2345',
-    'type': 'T_Query',
-    'P100': {
-      'type': 'Property',
-      'value': 12,
-      'observedAt': '2018-12-04T12:00:00',
-      'P1_R1': {
-        'type': 'Relationship',
-        'object': 'urn:ngsi-ld:T2:6789'
+    id: "urn:ngsi-ld:T_Query:EntityForQuery2345",
+    type: "T_Query",
+    P100: {
```

Comment:
it would be nice, copy-paste-wise, if JSON objects could be encoded with key terms between double quotes

@jason-fox is there any option in ESLint / Prettier that could do this for us?

for the avoidance of doubt, I mean

```
{
  "id": "urn:ngsi-ld:A:123",
  "type": "MyType",
}
```

etc.

Reply:
The simple answer is here is **NO** - prettier is an opinionated beautifier and it will apply a simple standard set of rules. The examples you are quoting are **object literals** not pure JSON and therefore the double quotes are not necessary in the code (even if they are necessary in a JSON file.

When copy-pasting, there are two ways around this.

1) Paste the JSON into the js code of the test with double quotes - when it is committed in git it will auto-format to the simpler object literal no-quote format.

2) Paste the JSON into a separate file and `require` it - add the location of JSON files to `.eslintignore`.